src/components/king-of-the-hill/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const formatPriceChange = (relativeChange?: number | null): string => {
+  if (!relativeChange && relativeChange !== 0) return 'N/A';
+
+  const percentage = (relativeChange * 100).toFixed(2);
+  const isPositive = relativeChange > 0;
+
+  // Use up arrow for positive, dash for negative with consistent spacing
+  return isPositive ? `↑ ${percentage}%` : `- ${Math.abs(parseFloat(percentage))}%`;
```

Comment:
this is a debatably good pattern, i didn't see off the top if we had some existing utilities here and perhaps could be done better with an object type, but wanted to pause before going too far in case i missed some existing helper or pattern we already have for this